src/Microsoft.PowerShell.Commands.Utility/commands/utility/new-object.cs

```diff
@@ -159,12 +159,13 @@ protected override void BeginProcessing()
                             NewObjectStrings.TypeNotFound,
                             TypeName);
 
-                        ThrowTerminatingError(
+                        WriteError(
```

Comment:
Above is still a AmbiguousTypeException - should we do the same where?

Reply:
The parent issue is closed, @iSazonov  what should be the next step ?